updating/updating-cluster-cli.adoc

```diff
@@ -15,6 +15,7 @@ See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permis
 * Ensure all Operators previously installed through Operator Lifecycle Manager (OLM) are updated to their latest version in their latest channel. Updating the Operators ensures they have a valid upgrade path when the default OperatorHub catalogs switch from the current minor version to the next during a cluster upgrade. See xref:../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Upgrading installed Operators] for more information.
 * Ensure that all machine config pools (MCPs) are running and not paused. Nodes associated with a paused MCP are skipped during the update process. You can pause the MCPs if you are performing a canary rollout update strategy.
 * If your cluster uses manually maintained credentials, ensure that the Cloud Credential Operator (CCO) is in an upgradeable state. For more information, see _Upgrading clusters with manually maintained credentials_ for xref:../installing/installing_aws/manually-creating-iam.adoc#manually-maintained-credentials-upgrade_manually-creating-iam-aws[AWS], xref:../installing/installing_azure/manually-creating-iam-azure.adoc#manually-maintained-credentials-upgrade_manually-creating-iam-azure[Azure], or xref:../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-maintained-credentials-upgrade_manually-creating-iam-gcp[GCP].
+* Ensure you are operating on a cluster operator that is upgradable so you can upgrade to minor releases without getting blocked.
```

Comment:
I think we need some guidance around how an admin can check this.

cc @sdodson

Reply:
Reading through OSDOCS-2477 and the epic it's linked to I would interpret this to just mean that the admin addresses any of the conditions triggered by Upgradeable=False displayed either by the Console or `oc adm upgrade` cli command. I'm sure how to best address that in docs, perhaps something like.

You must address any outstanding Upgradeable=False conditions before the cluster will allow an upgrade to the next minor. The output of `oc adm upgrade` will list all Upgradeable=False conditions and reasons for those conditions which will help you prepare for a minor version upgrade.